external/TypeParsing/System/Reflection/AssemblyNameFormatter.cs

```diff
@@ -0,0 +1,156 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
It might be better to match the file path as well (e.g. `src/System.Private.CoreLib/shared/System/Reflection/AssemblyNameFormatter.cs` in this case)